go.mod

```diff
@@ -24,10 +24,13 @@ require (
 	sigs.k8s.io/kind v0.15.0
 )
 
+require oras.land/oras-go v1.2.0
```

Comment:
can we include this with other `require`?

Reply:
:+1: